redfish-core/lib/oem/ibm/pcie_topology_refresh.hpp

```diff
@@ -46,6 +48,85 @@ inline void
         "PCIeTopologyRefresh");
 }
 
+/**
+ * @brief PCIe Topology Refresh monitor. which block incoming request
+ *
+ * @param[in] timer       pointer to steady timer which block call
+ * @param[in] aResp   Shared pointer for generating response message.
+ * @param[in] count   how many time this method get called.
+ *
+ * @return None.
+ */
+void pcIeTopologyRefreshWatchdog(
```

Comment:
nit: can we fix capitalization in pcIeTopologyRefreshWatchdog.

Reply:
ack